### PR TITLE
docs/variants: Modify/Add links to results on osfv-test-results at gh

### PR DESCRIPTION
**Pull request by philipandag**

### CODE REVIEW

**Comment by philipandag** [comment]
pre-commit.ci autofix

File: docs/variants/msi_z690/releases.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ For details about our release process please read
 ## v1.1.3 - 2024-01-22
 
 Test results for this release can be found
-[here](https://docs.google.com/spreadsheets/d/1wSE6xA3K3nXewwLn5lV39_2wZL1kg5AkGb4mvmG3bwE/edit#gid=1143764851).
+[here](https://github.com/Dasharo/osfv-results/tree/main/boards/NovaCumsistom/ms7d25/results.csv).
```
Comment (mkopec) [inline docs/variants/msi_z690/releases.md:22]:
```suggestion
[here](https://github.com/Dasharo/osfv-results/tree/main/boards/msi/ms7d25/results.csv).
```

File: docs/variants/msi_z790/releases.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ recommend seeing our [FAQ](../../unified/msi/faq.md).
 ## v0.9.1 - 2024-01-22
 
 Test results for this release can be found
-[here](https://docs.google.com/spreadsheets/d/1wSE6xA3K3nXewwLn5lV39_2wZL1kg5AkGb4mvmG3bwE/edit#gid=1753841105).
+[here](https://github.com/Dasharo/osfv-results/tree/main/boards/NovaCumsistom/ms7e06/results.csv).
```
Comment (mkopec) [inline docs/variants/msi_z790/releases.md:24]:
```suggestion
[here](https://github.com/Dasharo/osfv-results/tree/main/boards/msi/ms7e06/results.csv).
```

**Review by mkopec — CHANGES_REQUESTED** [review]
Looks good apart for 2 typos ;)

**Comment by philipandag** [comment]
Thanks for finding those errors. Interesing that I wrote `msi` inside `NovaCustom` and didn't notice that the result doesn't resemble what i wrote :)

**Review by mkopec — APPROVED** [review]

**Pull request merged**